AssetManagementTestProject/ActualData/UserActualData.cs

```diff
@@ -1,53 +1,53 @@
-﻿using CoreFramework.DriverCore;
-using OpenQA.Selenium;
-using AssetManagementTestProject.DAO;
+﻿//using CoreFramework.DriverCore;
+//using OpenQA.Selenium;
+//using AssetManagementTestProject.DAO;
 
-namespace AssetManagementTestProject.TestData;
-public class UserActualData : WebDriverAction
-{
-    public UserActualData() : base()
-    {
-    }
-    public UserDAO GetUserInfoFromGrid(string rowLocator, string cellLocator, int index)
-    {
-        // To check view user list
-        List<string> valuesFromCells = GetTextFromAllCellsOfOneRow
-            (rowLocator, cellLocator, index);
+//namespace AssetManagementTestProject.TestData;
+//public class UserActualData : WebDriverAction
```

Comment:
Why did you comment out code here? This should be used when writing test script for view user (Create a ViewUserDAO to store data got from the User List)